packages/eas-cli/src/project/customBuildConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { Platform } from '@expo/eas-build-job';
+import { BuildProfile } from '@expo/eas-json';
+import chalk from 'chalk';
+import fs from 'fs-extra';
+import path from 'path';
+
+export async function validateCustomBuildConfigAsync(
+  projectDir: string,
+  profile: BuildProfile<Platform>
+): Promise<void> {
+  if (!profile.config) {
+    return;
+  }
+
+  const relativeBuildConfigPath = getCustomBuildConfigPath(profile.config);
+  const buildConfigPath = path.join(projectDir, relativeBuildConfigPath);
```

Comment:
Why?

Reply:
I don't think we ever talked about it, but I always assumed this will be in root, like in every other CI I'm aware of.
On the other hand I don't have any other arguments, so if that was intentional I don't have any objections, but maybe post about it on slack before merging.

Reply:
this is an interesting discussion. i don't think we need to commit to any particular path right now, but maybe let's start off as keeping it as-is given that **eas.json** is also kept in the project directory. @dsokal - can you add a section about this to the design doc for us to have any further conversation there?

Reply:
This was a deliberate choice. As Brent noticed, eas.json is also in the project directory.